core/src/main/java/hudson/PluginWrapper.java

```diff
@@ -1306,10 +1306,16 @@ public HttpResponse doDoUninstall() throws IOException {
     public List<UpdateSite.Deprecation> getDeprecations() {
         /* Would be much nicer to go through getInfoFromAllSites but that only works for currently published plugins */
         List<UpdateSite.Deprecation> deprecations = new ArrayList<>();
-        for (UpdateSite site : Jenkins.get().getUpdateCenter().getSites()) {
```

Comment:
This line triggers a NPE when PluginDeprecationMonitor#isActivated is called. I've faced this on a support-core test, but I think it could be triggered as well if the  PluginDeprecationMonitor is enabled.

Reply:
The previous comment refers to [JENKINS-63562](https://issues.jenkins-ci.org/browse/JENKINS-63562).